test/sequential/test-debugger-list.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const fixtures = require('../common/fixtures');
+const startCLI = require('../common/debugger');
+
+const assert = require('assert');
+
+const cli = startCLI([fixtures.path('debugger/three-lines.js')]);
+
+cli.waitForInitialBreak()
```

Comment:
Can this be refactored into an async function with awaits rather than a long then chain?

Reply:
@jasnell done fea46820e4dc50e74340bc58e10c898bebfe7ed2